File: Juego.py
import random

def jugar():
    opciones = ['piedra', 'papel', 'tijera']

    victorias = 0
    derrotas = 0
    empates = 0

    while True:
        jugador_dos = random.choice(opciones)
        jugador = input("Elige piedra, papel o tijera (o 'salir' para terminar): ").lower()

        if jugador == 'salir':
            print("\n--- Resultado final ---")
            print(f"Victorias: {victorias}")
            print(f"Derrotas: {derrotas}")
            print(f"Empates: {empates}")
            print("¡Gracias por jugar! Hasta luego.")
            break

        if jugador not in opciones:
            print("Opción inválida. Intenta de nuevo.\n")
            continue

        print(f"Computadora eligió: {jugador_dos}")

        if jugador == jugador_dos:
            print("¡Empate!\n")
            empates += 1
        elif (jugador == 'piedra' and jugador_dos == 'tijera') or \
             (jugador == 'papel' and jugador_dos == 'piedra') or \
             (jugador == 'tijera' and jugador_dos == 'papel'):
            print("¡Ganaste!\n")
            victorias += 1
        else:
            print("¡Perdiste!\n")
            derrotas += 1


if __name__ == "__main__":
    jugar()

File: Juego.js
document.addEventListener("DOMContentLoaded", () => {
  const opciones = ["piedra", "papel", "tijera"];

  let victorias = 0;
  let derrotas = 0;
  let empates = 0;

  const victoriasEl = document.getElementById("victorias");
  const derrotasEl = document.getElementById("derrotas");
  const empatesEl = document.getElementById("empates");
  const jugadorEl = document.getElementById("jugador");
  const computadoraEl = document.getElementById("computadora");
  const mensajeEl = document.getElementById("mensaje");

  function jugar(jugador) {
    const computadora = opciones[Math.floor(Math.random() * opciones.length)];

    jugadorEl.textContent = jugador;
    computadoraEl.textContent = computadora;

    if (jugador === computadora) {
      mensajeEl.textContent = "¡Empate!";
      empates++;
    } else if (
      (jugador === "piedra" && computadora === "tijera") ||
      (jugador === "papel" && computadora === "piedra") ||
      (jugador === "tijera" && computadora === "papel")
    ) {
      mensajeEl.textContent = "¡Ganaste!";
      victorias++;
    } else {
      mensajeEl.textContent = "¡Perdiste!";
      derrotas++;
    }

    victoriasEl.textContent = victorias;
    derrotasEl.textContent = derrotas;
    empatesEl.textContent = empates;
  }

  // Agregar eventos a botones
  document.querySelectorAll(".choice").forEach(btn => {
    btn.addEventListener("click", () => jugar(btn.dataset.choice));
  });

    // Reiniciar el juego
  document.getElementById("reiniciar").addEventListener("click", () => {
    victorias = 0;
    derrotas = 0;
    empates = 0;

    victoriasEl.textContent = victorias;
    derrotasEl.textContent = derrotas;
    empatesEl.textContent = empates;
    jugadorEl.textContent = "—";
    computadoraEl.textContent = "—";
    mensajeEl.textContent = "Elige una opción para comenzar…";
  });
});
